static.nix

```diff
@@ -103,7 +103,10 @@ pkgs.mkShell ({
         zlib
         pcre
         openssl
-    ]);
+    ]) ++ pkgs.lib.optional withIOG (with pkgs; [ cddl cbor-diag ]
```

Comment:
Maybe add a one sentence comment explaining why it's a `buildInputs` rather than `nativeBuildInputs`, as it's slightly non-obvious to me! Otherwise, LGTM :)

Reply:
I'm not 100% sure about the cddl, and cbor-diag, however, for the other packages, if they are in the nativeBuildInputs, they won't end up in the pkg-config path.